app/views/items/lableSmall.pdf.erbtex

```diff
@@ -0,0 +1,25 @@
+\documentclass{article}
+\usepackage[utf8]{inputenc}
+\usepackage[T1]{fontenc}
+\usepackage[german]{babel}
+\usepackage[paperwidth=57mm,paperheight=32mm,nohead,nofoot,margin=5mm]{geometry}
```

Comment:
the only difference to lableBig.erbtex seems to be paperwith and paperheight arguments. maybe there are ruby variables more convenient, so we could define many formats in ruby and render them with the same LaTeX template.